crates/nu-protocol/src/engine/engine_state.rs

```diff
@@ -134,8 +125,8 @@ pub struct EngineState {
     pub previous_env_vars: HashMap<String, Value>,
     pub config: Config,
     pub pipeline_externals_state: Arc<(AtomicU32, AtomicU32)>,
-    pub repl_buffer_state: Arc<Mutex<Option<String>>>,
-    pub repl_operation_queue: Arc<Mutex<VecDeque<ReplOperation>>>,
+    pub repl_buffer_state: Arc<Mutex<Box<String>>>,
```

Comment:
I don't think we need to box the string, right?

Reply:
You're right, removed it. Thanks.